src/makielayout/blocks/polaraxis.jl

```diff
@@ -362,115 +860,99 @@ end
 ################################################################################
 
 
-function Makie.plot!(
-    po::PolarAxis, P::Makie.PlotFunc,
-    attributes::Makie.Attributes, args...;
+needs_tight_limits(::Surface) = true
```

Comment:
One set of test failures (basically everything surface related) comes from this, because this also affects `Axis`. Personally I think surface plots on an Axis defaulting to tight limits is something we want but I can also just revert this.